ios/ChangeNativeBridge.m

```diff
@@ -0,0 +1,35 @@
+//
+//  ChangeNativeBridge.m
+//  atb
+//
+//  Created by Mikael Brevik on 23/02/2021.
+//
+
+#import <Foundation/Foundation.h>
+#import "AppDelegate.h"
+#import "ChangeNativeBridge.h"
+#import <React/RCTLog.h>
+
+@implementation ChangeNativeBridge
+
+RCT_EXPORT_MODULE(ChangeNativeBridge);
+
+RCT_EXPORT_METHOD(changeAppearance: (NSString *)mode)
+{
+  if (@available(iOS 13.0, *)) {
+    dispatch_async(dispatch_get_main_queue(), ^{
+      
+      AppDelegate *appDelegate = (AppDelegate *)[UIApplication sharedApplication].delegate;
+      
+      if ([mode isEqualToString:@"dark"]) {
+        appDelegate.window.overrideUserInterfaceStyle = UIUserInterfaceStyleDark;
+      } else if ([mode isEqualToString:@"light"]) {
+        appDelegate.window.overrideUserInterfaceStyle = UIUserInterfaceStyleLight;
+      } else {
+        appDelegate.window.overrideUserInterfaceStyle = UIUserInterfaceStyleUnspecified;
+      }
```

Comment:
Something is not working as intended with this theme-logic. The text color on iOS is not correct when the system setting is dark mode, and the preference in the app is set to light mode.
<img width="410" alt="Screenshot 2021-02-24 at 11 28 42" src="https://user-images.githubusercontent.com/675421/108987689-dd4ccc80-7693-11eb-947c-92f4a2daca4d.png">

Reply:
This looks like old code? 😱 Have you pulled the latest changes?